mono-traversable/ChangeLog.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.10.1
```

Comment:
Please change this to 1.0.11.0

Reply:
I change the version bounds appropriately.

Reply:
I'm confused as to what you mean. I updated the haddock annotations and the `Changlog.md`. Is the current haddock annotations and `Changelog.md` comment what you were looking for?

Reply:
No, I was commenting that in the version I reviewed, the Haddocks and ChangeLog both still said 1.0.10.1.

Reply:
Oh, I understand what you mean now. I have updated the code and squashed the commits together.